### ISSUE TITLE
Does not work without GithubDektop

### ISSUE THREAD

**Issue by pilarArr-altran** [joyfullservice/msaccess-vcs-integration#18]
I get this message when I try to initialize the version control system

 Could not find GitHub Desktop software installation.

Why do I need GithubDektop to make this work?

**Comment by joyfullservice** [comment]
Hi @pilarArr-altran - This has been removed in the latest development version. I am not quite ready to roll it up to the master branch yet, but you can try out the latest version in the https://github.com/joyfullservice/msaccess-vcs-integration/tree/ConvertToAdd-in branch. This also makes the installation and usage much simpler because it doesn't require you to use the loader function or include the Version Control.accdb file with your project. It runs entirely from the add-in menu in Microsoft Access.

There is still a little work to do with some of the options for saving tables and obviously some updating of the documentation, but I have been making some good progress on the add-in over the past few weeks. Hopefully you will find this helpful with your project!

**Comment by pilarArr-altran** [comment]
Thanks for the answer @joyfullservice  
I will be trying the development branch 😃

**Comment by joyfullservice** [comment]
Good news! I was able to finish up the development yesterday to bring all the latest features into the master branch! This means you can download and use the file right from the main project on GitHub without needing to use the development branch.

**Comment by pilarArr-altran** [comment]
Hey @joyfullservice I saw yesterday that you merged the lastest release on master !! 🥇 
Unfortunately I work on a managed computer and I dont have admin priviledges to install the addIn.
I was sad because it looks just like the tool we need. If I ever get permissions to install it I'll check in with feedback. 
Thanks for sharing your work!

**Comment by joyfullservice** [comment]
Yes, I have been able to make some very good headway on the project this week. That has been a blessing!

In terms of installation, it only needs admin rights one time to add some registry keys to the HKLM\Software branch. (This is what Access uses to display the Add-In menu items.) After the initial installation, it doesn't need admin rights to update to the latest version going forward. (It would only need that if we changed the menu items.)

If it would be easier for you, I would be happy to provide you with a .reg file that includes the necessary keys. That might possibly be easier for your administrator to quickly review and install for you. Just let me know what version of Microsoft Access you are using (I.e. 2019 32-bit) and if the .reg file would be helpful.  😄
